nss-day-cohort-44/nutshell-jade-jackalopes
# Ps chat debug

# Description
Added chat features

Fixes # (issue)
## Type of change

# Testing Instructions
git fetch --all
git checkout ps-chat-debug
serve

chat should work and be visible across all users

# Checklist:
